week-3/F-array-forEach/exercise.js

```diff
@@ -6,9 +6,47 @@
 
   An array with numbers 0-15 has been provided.
 */
+//create a function that performs on each element in the array.
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(multiples(item){ 
+if (item %3 === 0){
+  return "Fizz";
+} else if (item %5 === 0){
+  return "Buzz";
+} else if (item %3 === 0 && item %5 === 0){
+  return "FizzBuzz";
+} else {
+  return item;
+}
+});
+
+console.log(item);
```

Comment:
`item` is not reachable from this line - it's enclosed in the `forEach` function - so I'm afraid this code doesn't run because `item` is undefined.